src/Sylius/Component/Core/Pricing/ChannelAndCurrencyBasedCalculator.php

```diff
@@ -49,14 +49,7 @@ public function calculate(PriceableInterface $subject, array $configuration, arr
         $channel = $context['channel'];
         $currencyCode = $context['currency'];
```

Comment:
This variables should be removed.
Is it needed to pass `$configuration` and `$context` if you only return price from `$subject`?